development_scripts.py

```diff
@@ -289,7 +289,7 @@ def transform_file(filepath):
     Example:
         >>> filepath = "tests/cisco_ios/show_version/cisco_ios_show_version.yml"
         >>> transform_parsed(filepath)
-        >>> 
+        >>>
```

Comment:
Do you mind removing the two files that aren't related to the templates?

`git checkout master development_scripts.py`

Reply:
Command ran.